### DISCUSSION TITLE
v20.9.0

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#4020] [Releases]
**Another small but important release, mostly oriented towards fixing a couple of bugs with the new UPS APIs.
Remember,  the old UPS XML APIs will be shut down on 3rd June 2024.**

## What's Changed
* Fixed unable to select "free method" in UPS backend configuration by @fballiano in https://github.com/OpenMage/magento-lts/pull/4005
* Fixed erroneous call to getText() method in Mage_Usa_Model_Shipping_Carrier_Ups::getAllowedMethods() by @ragnese in https://github.com/OpenMage/magento-lts/pull/4013
* Fixed available currencies filtering in UPS _parseRestResponse() method by @fballiano in https://github.com/OpenMage/magento-lts/pull/4017
* Added "edit" action column to "Associated Products" grid of grouped products by @empiricompany in https://github.com/OpenMage/magento-lts/pull/4006
* Fixed customer can't login after registering through PayPal Express Checkout by @fballiano in https://github.com/OpenMage/magento-lts/pull/3979
* Added filemtime to merged JS/CSS hash calculation algorithm by @boesbo in https://github.com/OpenMage/magento-lts/pull/4004
* Added stopPropagation to "copy text" buttons in backend by @Caprico85 in https://github.com/OpenMage/magento-lts/pull/4019
* Updated PHPStan to 1.11.2 by @fballiano in https://github.com/OpenMage/magento-lts/pull/3988
* Bump squizlabs/php_codesniffer from 3.9.2 to 3.10.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4011
* Bump friendsofphp/php-cs-fixer from 3.57.1 to 3.57.2 by @dependabot in https://github.com/OpenMage/magento-lts/pull/4012

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMage/magento-lts/releases/tag/v20.9.0'>v20.9.0</a>.</em>

**Comment by toejam34** [comment]
Upgraded flawlessly again. 

Thanks team for your continual work.
